### ISSUE TITLE
about document proportions

### ISSUE THREAD

**Issue by ShellingFord221** [meereeum/lda2vec-tf#12]
Hi, Figure 1 in the paper shows that there first comes document weights, then after a softmax layer, there comes document proportions. But in the loss function, there is no document weights, only the document proportion p_jk. So are document proportions parameters in the network, or are they just learned by other methods beforehand? Thanks!
